### ISSUE TITLE
和论文中不同的地方

### ISSUE THREAD

**Issue by dl8207531** [xggIoU/centernet_tensorflow_wilderface_voc#15]
请问下这里没用到论文里提到的高斯核和dcn吗？

**Comment by xggIoU** [comment]
用到了高斯核，以及自己改的高斯核。没有采用dcn。

**Comment by dl8207531** [comment]
`
output_center_peak=tf.layers.max_pooling2d(output_center,3,1,padding='same')
peak_mask=tf.cast(tf.equal(output_center,output_center_peak),tf.float32)
thresh_mask=tf.cast(tf.greater(output_center,class_prob_thresh),tf.float32)
obj_mask=peak_mask*thresh_mask
output_center=output_center*obj_mask
`
看到了 是这个吧 
谢谢

**Comment by xggIoU** [comment]
这个是最后的预测热图decode操作。为了得到目标的中心点。

**Comment by dl8207531** [comment]
是这个函数吗？ gaussian_radius 没找到哪里调用 在生成数据的时候使用到的？

**Comment by xggIoU** [comment]
#heatmap_roi = creat_roiheatmap_circle(center_map_obj, size_map_int)
 heatmap_roi = creat_roiheatmap_ellipse(center_map_obj, size_map_int)

**Comment by dl8207531** [comment]
了解，谢谢
